src/main/java/uk/gov/ons/ctp/response/collection/exercise/client/impl/ActionSvcRestClientImpl.java

```diff
@@ -92,9 +90,8 @@ public ActionPlanDTO createActionPlan(
 
     ActionPlanDTO createdActionPlan =
         restTemplate.postForObject(uriComponents.toUri(), httpEntity, ActionPlanDTO.class);
-    log.debug(
-        "Successfully posted to action service to create action plan, ActionPlanId: {}",
-        createdActionPlan.getId());
+    log.with("ActionPlanId", createdActionPlan.getId())
```

Comment:
It would be good to get some agreement on the format of keys. I went for snake case to be consistent with the python services

Reply:
Makes sense i'll change to snake case saves us changing the python services.

Reply:
I dun snake_case 2